src/heltour.js

```diff
@@ -1,16 +1,20 @@
 //------------------------------------------------------------------------------
 // Heltour related facilities
 //------------------------------------------------------------------------------
-
+const _ = require('lodash');
 const url = require('url');
 const http = require("./http.js");
 
-function findPairing(heltourConfig, white, black) {
+function findPairing(heltourConfig, white, black, league_tag) {
```

Comment:
Technically, `heltourConfig` has a league_tag in it. it comes from the config. I am unsure about whether we should use it or explicitly pass in the league tag.  I guess explicitly passing it in allows us to _not_ pass it in when we want to get pairings for both leagues.  

Let's document that here.